src/ban.cpp

```diff
@@ -120,24 +130,23 @@ std::string BanManager::getBanName(const std::string &ip)
 void BanManager::add(const std::string &ip, const std::string &name)
 {
 	MutexAutoLock lock(m_mutex);
-	m_ips[ip] = name;
+	m_ips[normalizeIp(ip)] = name;
 	m_modified = true;
 }
 
 void BanManager::remove(const std::string &ip_or_name)
 {
+	std::string norm = normalizeIp(ip_or_name);
```

Comment:
it also "normalizes" the name. So if a good player's name is "NyanKitty", and the bad player "nyankitty" gets banned, and the admin unbans NyanKitty, does it then also unban nyankitty?